### PR TITLE
Remove useless materialized views

### PR DESCRIPTION
**Pull request by piotrekpasciak**

### CODE REVIEW

File: app/policies/leaderboard_policy.rb [hunk 1]
```diff
@@ -76,6 +77,7 @@ def resolve
           ]
           scope.where(sql)
         else
+          binding.pry
```
Comment (nemaniarjun) [inline app/policies/leaderboard_policy.rb]: Do we need these debugging statements ?

Comment (piotrekpasciak) [reply app/policies/leaderboard_policy.rb]: Definitely not, I forgot to remove them. PR is in work-in-progress state anyway.

**Comment by nemaniarjun** [comment]
Once https://github.com/AIcrowd/AIcrowd/pull/1126 is merged we can also remove the challenge_round_summary view.

**Comment by nemaniarjun** [comment]
Now that #1126 is merged, we are not using the challenge_round_summary view anywhere in the codebase. we should remove that view as well in this PR. :smile:

**Pull request closed without merging**